packages/@aws-cdk/aws-backup/lib/vault.ts

```diff
@@ -15,6 +15,19 @@ export interface IBackupVault extends IResource {
    * @attribute
    */
   readonly backupVaultName: string;
+
+  /**
+   * The ARN of the backup vault.
+   *
+   * @attribute
+   */
+  readonly backupVaultArn: string;
+
+  /**
+   * Grant the actions defined in actions to the given grantee
+   * on this Backup Vault resource.
+   */
+  grant(grantee: iam.IGrantable, ...actions: string[]): iam.Grant;
```

Comment:
are there any common permissions patterns that you've used with backup vaults that would warrant a more specific grant method?

Reply:
So far, it was just trigger a backup job. it's not anything complex (single action) so I would leave it as is for now.